app/edge/devices/components/EdgeDevicesDatatable/EdgeDevicesDatatableActions.tsx

```diff
@@ -0,0 +1,125 @@
+import { useRouter } from '@uirouter/react';
+
+import type { Environment } from '@/portainer/environments/types';
+import { Button } from '@/portainer/components/Button';
+import { confirm } from '@/portainer/services/modal.service/confirm';
+import { prompt } from '@/portainer/services/modal.service/prompt';
+import * as notifications from '@/portainer/services/notifications';
+import { activateDevice } from '@/portainer/hostmanagement/open-amt/open-amt.service';
+
+interface Props {
+  selectedItems: Environment[];
+  isFDOEnabled: boolean;
+  isOpenAMTEnabled: boolean;
+  setLoadingMessage(message: string): void;
+}
+
+export function EdgeDevicesDatatableActions({
+  selectedItems,
+  isOpenAMTEnabled,
+  isFDOEnabled,
+  setLoadingMessage,
+}: Props) {
+  const hasSelectedItem = selectedItems.length === 1;
```

Comment:
I know it's product/design decision, but if it's only for one item, why not put it in a table column?

Reply:
I believe that the idea is to be able to support multiple items in the future, doing it this way makes it much easier to iterate the feature